diff --git a/Dockerfile-8.x b/Dockerfile-8.x
index 8eda1ae..52c776e 100644
--- a/Dockerfile-8.x
+++ b/Dockerfile-8.x
@@ -34,12 +34,25 @@ RUN     yum install -y \
             yum-utils \
             epel-release
 
+# Building edk2
+RUN yum install -y \
+	libuuid-devel \
+	nasm \
+	iasl \
+	ipxe-efi
+
 # Niceties
 RUN     yum install -y \
             vim \
             wget \
             which
 
+# Dom0 libs
+RUN     yum install -y \
+	xen-dom0-libs-devel \
+    openssl-devel \
+    libxml2-devel
+
 # OCaml in XS may be older than in CentOS
 RUN     sed -i "/gpgkey/a exclude=ocaml*" /etc/yum.repos.d/Cent* /etc/yum.repos.d/epel*
 
@@ -52,3 +65,4 @@ RUN     useradd builder \
 RUN     mkdir -p /usr/local/bin
 COPY    files/init-container.sh /usr/local/bin/init-container.sh
 COPY    files/rpmmacros /home/builder/.rpmmacros
+
